packages/components/src/pages/Quickstart.vue

```diff
@@ -0,0 +1,571 @@
+<template>
+  <div class="qs">
+    <div class="qs-container">
+      <div class="qs-head">
+        <div class="qs-head__btn-wrap">
+          <button
+            class="qs-head__btn"
+            type="button"
+            v-if="currentStep !== 1"
+            @click="prevStep"
+          >
+            <StepLeftIcon class="qs-head__btn-icon" />
+          </button>
+        </div>
+        <span class="qs-head__step"
+          >step {{ currentStep }}/{{ stepsCount }}</span
+        >
+        <div class="qs-head__btn-wrap">
+          <button
+            class="qs-head__btn"
+            type="button"
+            v-if="currentStep !== stepsCount"
+            @click="nextStep"
+          >
+            <StepRightIcon class="qs-head__btn-icon" />
+          </button>
+        </div>
+      </div>
+      <section class="qs-step" v-if="currentStep === 1">
+        <div class="qs-step__head">
+          <h1 class="qs-title">Install AppMap Agent</h1>
+          <select class="qs-select" v-model="language">
+            <option
+              v-for="lang in Object.keys(languages)"
+              :key="lang"
+              :value="lang"
+            >
+              {{ languages[lang] }}
+            </option>
+          </select>
+        </div>
+        <div class="qs-step__block" v-if="!step1Completed">
+          <p>
+            This will add the following snippet to the top of your Gemfile and
+            run bundle to install the AppMap gem.
+          </p>
+          <code class="qs-code" @click="select"
+            >gem "appmap", "= 0.53.0", :groups => [:development, :test]</code
```

Comment:
How can we specify this text for the other languages (Python, Java)? We may need to break this component up into smaller components so that we may reuse them with minor content differences.